packages/adapter-next/package.json

```diff
@@ -74,13 +74,13 @@
 		"@types/prettier": "^2.7.2",
 		"@types/react": "^18.0.28",
 		"@types/semver": "^7.3.13",
-		"@typescript-eslint/eslint-plugin": "^5.55.0",
+		"@typescript-eslint/eslint-plugin": "~5.55.0",
```

Comment:
I would not change that in this MR since the CI is flaky, best is to change how we are doing (and super soon) I think Baptiste wants to give us a plan for that.

Reply:
should have been reverted

Reply:
What do you mean?

Reply:
I thought those changes where reverted by now :/